instance/github.py

```diff
@@ -157,6 +170,20 @@ def __init__(self, number, fork_name, branch_name, title, username, body=''):
         self.body = body
 
     @property
+    def truncated_title(self):
+        """
+        This PR's title truncated to 4 words
+        """
+        return truncatewords(self.title, 4)
+
+    @property
+    def sub_domain(self):
+        """
+        The subdomain for this PR
+        """
+        return 'pr{number}.sandbox'.format(number=self.number)
```

Comment:
@omarkhan Similar remark to https://github.com/open-craft/opencraft/pull/11#discussion_r41085163 - PRs might not all be for sandboxes, so I'd rather stay clear of adding logic specific to sandboxes there too, at least for now. Can we keep the choice of domain name in the task that creates the instance?